distribution/sc-proxy-dex/src/proxy_pair.rs

```diff
@@ -561,13 +586,27 @@ pub trait ProxyPairModule: proxy_common::ProxyCommonModule {
     }
 
     #[view(getTemporaryFunds)]
+    fn get_temporary_funds(
+        &self,
+        address: &Address,
+    ) -> MultiResultVec<GenericEsdtAmountPair<Self::BigUint>> {
+        let vec: Vec<GenericEsdtAmountPair<Self::BigUint>> = self
+            .temporary_funds(address)
+            .iter()
+            .map(|x| GenericEsdtAmountPair {
+                token_id: x.0 .0,
+                token_nonce: x.0 .1,
+                amount: x.1,
```

Comment:
It's okay as it is but could be made a little easier to read like this
```suggestion
                let (key, amount) = x;
                let (token_id, token_nonce) = key;
                GenericEsdtAmountPair { token_id, token_nonce, amount }
```

Reply:
Didn't know about this syntax. Thank you. Fixed